lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -183,11 +183,14 @@ private Pagination fromSearch(kbasesearchengine.search.Pagination pg) {
             ret.objectInfo = true;
             ret.objectData = true;
             ret.objectKeys = true;
+            ret.objectHighlight = false;
         } else {
             boolean idsOnly = toBool(pp.getIdsOnly());
             ret.objectInfo = !(toBool(pp.getSkipInfo()) || idsOnly);
             ret.objectData = !(toBool(pp.getSkipData()) || idsOnly);
             ret.objectKeys = !(toBool(pp.getSkipKeys()) || idsOnly);
+            //default to false currently b/c of search tags. TODO: add search tags to black list?
+            ret.objectHighlight = toBool(pp.getIncludeHighlight()) && !idsOnly;
```

Comment:
This line is only partially covered

Reply:
you mean also check with idsOnly?

Reply:
Dunno. Whatever getting full coverage of the line takes.